backend/src/models/service.ts

```diff
@@ -16,22 +16,22 @@ import { Scan } from './scan';
 const filterProducts = (product: Product) => {
   // Filter out false positives.
   const { cpe, version } = product;
-  if (cpe === 'cpe:/a:apache:tomcat' && version === '1.1') {
+  if (cpe?.includes('apache:tomcat') && version === '1.1') {
     // Wappalyzer incorrectly detects "Apache Tomcat 1.1"
     // https://github.com/AliasIO/wappalyzer/issues/3305
     return false;
   }
-  if (cpe === 'cpe:/a:apache:coyote:1.1:') {
+  if (cpe?.includes('apache:coyote') && version === '1.1') {
     // Intrigue Ident incorrectly detects "Apache Coyote 1.1"
     // https://github.com/intrigueio/intrigue-ident/issues/51
     return false;
   }
-  if (cpe === 'cpe:/a::generic:unauthorized::') {
+  if (cpe?.includes('generic:unauthorized')) {
     // Intrigue Ident sometimes detects "Unauthorized" CPEs
     return false;
   }
   if (
-    cpe === 'cpe:/a:f5:big-ip_application_security_manager:14.0.0_and_later:'
+    cpe?.includes('f5:big-ip_application_security_manager:14.0.0_and_later')
```

Comment:
What exactly was the issue here? I'll add a test with the cpe that was failing so we make sure it doesn't fail again.

Reply:
The unauthorized one wasn't getting flagged, since the actual version was `cpe:/:generic:unauthorized::` (slightly different from what we had). Changed all of these to include so we're not quite as strict